### ISSUE TITLE
Diverting Respons messages

### ISSUE THREAD

**Issue by WieserJulian** [rechen-werk/Kilian#26]
I stumpled over it while testing some features of kilian.
If you use kusss the first time the response makes perfect sense but if you use it twice shouldnt it be an update more then just a complete new student.  (response on discord)
Also on  /unkusss if u arent in the database shouldnt it tell u so?

**Comment by rechen-werk** [comment]
Yeah you're right, on `/unkusss` it should definitely tell you that youre not in the system.
On `/kusss` it would be cool too if it doesnt bloat the code up too much. ;)

**Comment by WieserJulian** [comment]
should i implement an comunication handler for any later purpose or fix implement it into `/kusss` and `/unkusss`

**Comment by rechen-werk** [comment]
i think just fixing it in the corresponding methods is ok, don't over engineer it :P
